js/api.md

```diff
@@ -2212,18 +2212,29 @@ postData();
 
 ## Using Modular Imports
 
-If you only need to use API, you can run: `npm install @aws-amplify/api` which will only install the API module.
-Note: if you're using Cognito Federated Identity Pool to get AWS credentials, please also install `@aws-amplify/auth`.
-Note: if you're using Graphql, please also install `@aws-amplify/pubsub`
+Amplify V3 now supports REST and GraphQL modular imports such as if you only need to use REST API,
+you can run: `npm install @aws-amplify/api-rest` which will only install the REST API module and
+reduce your application bundle size by not including any GraphQL related modules.
 
-Then in your code, you can import the Api module by:
+Then in your code, you can import this module by:
 
 ```javascript
-import API, { graphqlOperation } from '@aws-amplify/api';
+import RestAPI from '@aws-amplify/api-rest';
+
+RestAPI.configure();
+```
+
+Similarly to use only the GraphQL API you can run: `npm install @aws-amplify/api-graphql` and then in your code:
 
-API.configure();
+```javascript
+import GraphqlAPI, { graphqlOperation } from '@aws-amplify/api-graphql';
```

Comment:
Suggestion to match https://github.com/aws-amplify/amplify-js/issues/3365#issuecomment-502009495:

```suggestion
import GraphQLAPI, { graphqlOperation } from '@aws-amplify/api-graphql';
```